rust/kernel/file_operations.rs

```diff
@@ -29,10 +23,6 @@ impl File {
     pub fn pos(&self) -> u64 {
         unsafe { (*self.ptr).f_pos as u64 }
     }
-
-    pub fn flags(&self) -> FileFlags {
```

Comment:
Is there a future plan for this? It's useful to be able to access flags :-)

Reply:
Indeed, `bitflags` was the last thing with a dependency (and was not used at the moment), so I decided to take it out. The options forward for crates that we might want to use can be: come up with a replacement (possibly one more suited for the kernel) or adapt a particular version of the code (like I did with `shlex`).